packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -352,6 +352,9 @@ export const iconMetadata: {
     IconMobile: {
         tags: ['phone']
     },
+    IconNi: {
+        tags: ['National Instruments', 'logo', 'product']
```

Comment:
😝
```suggestion
        tags: ['National Instruments', 'Emerson Test and Measurement Business Group', 'logo', 'product', ]
```

Reply:
I'd probably not actually do this since I think it'll clip / be annoyingly long in the metaphor column lol. Or if done make it last so the product and logo metaphors aren't clipped

Reply:
I agree, just a silly joke for now.